build.gradle

```diff
@@ -41,6 +41,12 @@ subprojects {
 
     sourceCompatibility = 1.7
 
+    tasks.withType(Javadoc) {
+        // suppress Javadoc doclint warnings in Java 8+
+        if (!System.getProperty("java.version").startsWith("1.7")) {
```

Comment:
How so? Right now source compatibility is set to be Java 1.7 and CircleCI runs the build using Java 7.